### PR TITLE
netdiscover 0.3-pre-beta7 (new formula)

### PR DESCRIPTION
**Pull request by mtlott**

### CODE REVIEW

File: Library/Formula/netdiscover.rb [hunk 1]
```diff
@@ -0,0 +1,185 @@
+class Netdiscover < Formula
+  desc "ARP Scanner"
+  homepage "http://sourceforge.net/projects/netdiscover/"
+  url "svn://svn.code.sf.net/p/netdiscover/code/trunk"
+  version "0.3-pre-beta7"
```
Comment (dunn) [inline Library/Formula/netdiscover.rb]: I'm afraid it'll have to go to our [devel-only tap](https://github.com/Homebrew/homebrew-devel-only) unless there's a stable version.

File: Library/Formula/netdiscover.rb [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gawk" => :build
+  depends_on "libnet"
+
+  # In order to get this working on OS X some changes were needed.
+  # While I will try to get this accepted upstream, it is not clear
+  # if the author is still maintaining this. These changes work against
```
Comment (dunn) [inline Library/Formula/netdiscover.rb]: We're going to be reluctant to accept this if it needs patching to work on OSX and it's not still being maintained.

**Comment by mtlott** [comment]
This is one of a hundreds of tools which are readily available through Linux package managers but are missing in any form on OS X. Hundreds of tools in the Linux world are not "actively" maintained. They were written. They do there job. You can recompile them ad-nauseum and they still just function. This is one of them. Written 10 years ago and patched/changed maybe once a year since. It just works under Linux. OS X is different animal for various reasons.

It is a lot of work to get these tools to function on OS X in the least invasive method possible. Having just spent a lot of time on this one (and trying to get others to work as well). I would hope to have that knowledge captured somewhere such that some one else does not have to go through the 30+ hours I just did on this one tool.

Don't let the version number fool you. People version their tools whatever fancies them at the time. This tool has never had a non-beta version moniker. Probably just the authors way of saying use at your own risk. I mean seriously, beta for 10 years? This has not stopped the Linux world from including it in all of the major distributions for as long as I can remember.

Incidentally, the major issue in getting this to work seems to be a change in El Capitan's libpcap library.

If you are not interested in these ports (which don't exist on other sites either) just let me know. It takes long enough to get the code working in the first place. I can save some time by not sharing the work.

**Comment by mikemcquaid** [comment]
> It is a lot of work to get these tools to function on OS X in the least invasive method possible. Having just spent a lot of time on this one (and trying to get others to work as well). I would hope to have that knowledge captured somewhere such that some one else does not have to go through the 30+ hours I just did on this one tool.

I'm sorry that you've spent that time and we're not going to include it. For future reference this may be helpful: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#stable-versions

To share your work you may want to consider [creating your own tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) instead. Thanks!

**Pull request closed without merging**